cognite/client/experimental/analytics/models.py

```diff
@@ -0,0 +1,371 @@
+# -*- coding: utf-8 -*-
+from typing import Any, Dict, List
+
+import requests
+
+from cognite.client._api_client import APIClient
+
+
+class ModelsClient(APIClient):
+    def __init__(self, **kwargs):
+        kwargs["version"] = "0.6"
+        super().__init__(**kwargs)
+
+    def create_model(
+        self,
+        name: str,
+        description: str = "",
+        metadata: Dict[str, Any] = None,
+        input_fields: List[str] = None,
+        output_fields: List[str] = None,
+    ) -> Dict:
+        """Creates a new model
+
+        Args:
+            name (str):             Name of model
+            description (str):      Description
+            metadata (Dict[str, Any]):          Metadata about model
+            input_fields (List[str]):   List of input fields the model accepts
+            output_fields (List[str]:   List of output fields the model produces
+
+        Returns:
+            Dict: The created model.
+        """
+        url = "/analytics/models"
+        model_body = {
+            "name": name,
+            "description": description,
+            "metadata": metadata or {},
+            "inputFields": input_fields or [],
+            "outputFields": output_fields or [],
+        }
+        res = self._post(url, body=model_body)
+        return res.json()
+
+    def get_models(self) -> List[Dict]:
```

Comment:
Completely agree. Was just lazy here because I knew I would have to change stuff here in the future. I think it is not necessary to be too strict on that stuff when functionality is added to the "experimental client".

Reply:
Make sense for the experimental client. We should probably prioritise adding them for the other stable endpoints though in the next release (data objects with defined fields instead of general response objects that is). The response objects are basically just a wrapper where you call to_json() to get the same dictionary.

Reply:
Ah yes. Agree.